samples/PipeCI.Blob.Sample/Startup.cs

```diff
@@ -39,14 +39,15 @@ public void Configure(IApplicationBuilder app, ILoggerFactory logger)
 
             app.Run(async (context) =>
             {
-                await context.Response.WriteAsync(@"<html>
-    <body>
-        <form action=""/blob/upload"" method=""post"" enctype=""multipart/form-data"">
-            <input type=""file"" name=""file"" />
-            <input type=""submit"" />
-        </form>
-    </body>
-</html>");
+                await context.Response.WriteAsync(@"
+                                                    <html>
+                                                        <body>
+                                                            <form action=""/blob/upload"" method=""post"" enctype=""multipart/form-data"">
+                                                                <input type=""file"" name=""file"" />
+                                                                <input type=""submit"" />
+                                                            </form>
+                                                        </body>
+                                                    </html>");

```

Comment:
See the things which Microsoft did. (https://github.com/aspnet/Localization/blob/dev/samples/LocalizationSample/Startup.cs#L79-L102)